client/src/partials/fiscal/fiscal.html

```diff
@@ -27,11 +27,23 @@
           {{ 'FISCAL.CREATE_FISCAL_YEAR' | translate }}
         </a>
       </div>
+      <div
+        class="toolbar-item"
+        ng-if="FiscalCtrl.state.current.name !== 'fiscal.list'">
```

Comment:
Would it be possible to store these in a variable?  Something like `FiscalCtrl.isListState`?